test/e2e/crud.go

```diff
@@ -111,11 +116,16 @@ var _ = Describe("Federated", func() {
 					tl.Fatalf("Error creating federated resource: %v", err)
 				}
 				hostClient := genericclient.NewForConfigOrDie(f.KubeConfig())
-				defer hostClient.Delete(context.TODO(), createdObj, createdObj.GetNamespace(), createdObj.GetName())
+				defer func() {
+					err := hostClient.Delete(context.TODO(), createdObj, createdObj.GetNamespace(), createdObj.GetName())
+					if err != nil {
+						tl.Fatalf("Unexpected error: %v", err)
+					}
+				}()
 
 				waitDuration := 10 * time.Second // Arbitrary amount of time to wait for deletion
 				By(fmt.Sprintf("Checking that the unlabeled resource is not deleted within %v", waitDuration))
-				wait.PollImmediate(framework.PollInterval, waitDuration, func() (bool, error) {
+				err = wait.PollImmediate(framework.PollInterval, waitDuration, func() (bool, error) {
```

Comment:
Ignoring this error was intentional, as timing out is success, and it should continue to be ignored (e.g. `_ = wait.PollImmediate`).

Reply:
Note that this change is responsible for the CI failure.